src/Raven.Server/Documents/Replication/Senders/MergedEnumerator.cs

```diff
@@ -35,7 +35,7 @@ public virtual bool MoveNext()
             var current = _workEnumerators[0];
             for (var index = 1; index < _workEnumerators.Count; index++)
             {
-                if (_comparer.Compare(_workEnumerators[index].Current, current.Current) > 0)
+                if (_comparer.Compare(_workEnumerators[index].Current, current.Current) < 0)
```

Comment:
What is this change about? Doesn't seem related to the PR

Reply:
`MergedEnumerator` is inherited only by the `MergedReplicationBatchEnumerator` which has its own implementation of `MoveNext`. So the `MoveNext` from the `MergedEnumerator` isn't used at all.
I'll also change it in previous versions.

Reply:
Actually, it was introduced in 6.0.